modin/pandas/test/test_io.py

```diff
@@ -483,6 +511,11 @@ def test_read_csv_datetime(
                 "The reason of tests fail in `cloud` mode is unknown for now - issue #2340"
             )
 
+        if Backend.get() == "Omnisci" and not isinstance(parse_dates, bool):
+            pytest.xfail(
+                "In some cases read_csv with `parse_dates` with OmniSci backend outputs incorrect result"
```

Comment:
submit a tracking issue and reference it here

Reply:
Fixed.